clients/venice-thin-client/src/test/java/com/linkedin/venice/client/stats/BasicClientStatsTest.java

```diff
@@ -156,6 +158,15 @@ public void testEmitUnhealthyRequestMetricsForDavinciClientWithWrongClientType()
     Assert.assertFalse(metrics.get(".test_store--request.OccurrenceRate").value() > 0.0);
   }
 
+  @Test
+  public void testEmitRequestRetryMetrics() {
+    InMemoryMetricReader inMemoryMetricReader = InMemoryMetricReader.create();
```

Comment:
Missed this last time. Can we use this to verify the metrics?

Reply:
I assume you meant for the OTel metics verification, as the Tehuti one is there. Updated with more unit test coverage.